### ISSUE TITLE
Marketplace: In the manage sites modal of global plugins, the different actions are confusing

### ISSUE THREAD

**Issue by ebinnion** [Automattic/wp-calypso#92252]
### Quick summary

![CleanShot 2024-07-01 at 12 49 02@2x](https://github.com/Automattic/wp-calypso/assets/1126811/42d87fc8-f2c9-44b7-bc11-ea2aaf1041fe)

There are `Install` and `Go to plugins page` actions on the manage sites modal and the difference between these is unclear. In the screenshot above, for example, each of those sites is a WoA site.

### Steps to reproduce

1. Go to `/plugins`
2. Click on a plugin
3. Click on `Manage Sites`
4. Observe different action button options

### What you expected to happen

A clearer UX/understanding of what the actions are and why they're present.

### What actually happened

I'm confused about why I can install on some sites but have to visit an interstitial page on other sites.

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

**Comment by mmtr** [comment]
> There are Install and Go to plugins page actions on the manage sites modal and the difference between these is unclear. In the screenshot above, for example, each of those sites is a WoA site.

I wonder if this is because the last two sites are ephemeral / trial / staging sites. 

For my sites, the actions are consistent to the same site type but I don't have ephemeral / trial / staging sites:
- For Simples sites is "Go to plugins page"
- For Atomic sites is "Install"

But yeah, I think we can do a better job here. For simples sites it should be either "Upgrade" (it the site does not have the right plan) or "Install" (if the site has the needed plan but has not been transferred to Atomic yet).

**Comment by javierarce** [comment]
I'm not sure why, but on some plugins (https://wordpress.com/plugins/js-composer) I see these different buttons:

![Image](https://github.com/user-attachments/assets/4c2fc23b-6a45-4ef3-83e5-82b30fe7cbc7)

**Comment by mmtr** [comment]
> I see these different buttons

Possibly a bug with paid plugins (aka 3rd-party or marketplace plugins). 

- If the site has a Business plan or higher, it should say "Purchase and activate"
- If the site has a Premium plan or lower, it should say "Upgrade and activate"

But it seems that all your sites in that screenshot had a Business plan (because all are `.wpcomstaging.com` domains), so all of them should display a "Purchase and activate" button. So yeah, likely we have an issue somewhere.

**Comment by javierarce** [comment]
> But it seems that all your sites in that screenshot had a Business plan (because all are .wpcomstaging.com domains), so all of them should display a "Purchase and activate" button. So yeah, likely we have an issue somewhere.

The same thing happens with free sites 😅 

![CleanShot 2024-08-22 at 10 26 57@2x](https://github.com/user-attachments/assets/09064d61-e634-4814-b6bf-aad1b11d3d91)

**Comment by mmtr** [comment]
I love that! Looks similar to what @lucasmendes-design proposed but as a full page instead of a modal (p9Jlb4-dqu-p2).

Maybe we can work on a i3 of the site selector project and include this use case as well.

**Comment by davemart-in** [comment]
I really like this proposal as well. Nicely done @javierarce! I'd say let's move this direction.

**Comment by lucasmendes-design** [comment]
Nicely done! 🌟

Some thoughts: 

- Do we need a badge "Requires plan upgrade" so prominent? Maybe the same copy with yellow 50. 
- After clicking the install button, the next page is the checkout page, correct? 

:)

**Comment by lucasmendes-design** [comment]
> I love that! Looks similar to what @lucasmendes-design proposed but as a full page instead of a modal (p9Jlb4-dqu-p2).

Maybe we can work on a i3 of the site selector project and include this use case as well.

+1 :)

**Comment by javierarce** [comment]
@lucasmendes-design 

> Do we need a badge "Requires plan upgrade" so prominent? Maybe the same copy with yellow 50.

You are totally right, that could work and it's also more consistent with the styles we have in other tables (eg: the domains table.)

![image](https://github.com/user-attachments/assets/27969151-80ee-48e3-bbc8-27f78109438e)

> After clicking the install button, the next page is the checkout page, correct?

For the plugins that require an upgrade, yes.
